_help/index.md

```diff
@@ -31,12 +33,13 @@ links:
       <ul class="pl0">
       {% for subpage in page.links %}
         {% assign subpage_slug = subpage.name | slugify %}
-        {% assign first_subpage = site.help | where_exp: "item", "item.url contains subpage_slug" | where: 'order', 1 | first %}
+        {% assign lang_path = "/" | append: site.lang | append: "/" %}
+        {% assign first_subpage = site.help | where_exp: "item", "item.url contains lang_path" | where_exp: "item", "item.url contains subpage_slug" | where: 'order', 1 | first %}
```

Comment:
Just curious, why change the url to `/en/` here but not in other places? Because it is the help center?